### PR TITLE
solve struct literal uses unkeyed fields

### PR DESCRIPTION
**Pull request by maoqide**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
 - solve complaints of `struct literal uses unkeyed fields` when go vet.

#### Which issue(s) this PR fixes:

Fixes # NONE

#### Does this PR introduce a user-facing change?

NONE

```release-note

```

### CODE REVIEW

**Comment by ffromani** [comment]
/ok-to-test

hi, thanks for this contribution! since the project CI passes cleanly so far, which run of `go vet` highlighted these issues?

**Comment by maoqide** [comment]
> /ok-to-test
> 
> hi, thanks for this contribution! since the project CI passes cleanly so far, which run of `go vet` highlighted these issues?

running `go vet ./...` under project root directory, output like:

# sigs.k8s.io/scheduler-plugins/pkg/networkaware/networkoverhead
# [sigs.k8s.io/scheduler-plugins/pkg/networkaware/networkoverhead]
pkg/networkaware/networkoverhead/networkoverhead_test.go:289:29: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/networkoverhead/networkoverhead_test.go:290:29: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/networkoverhead/networkoverhead_test.go:334:23: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/networkoverhead/networkoverhead_test.go:334:73: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
# sigs.k8s.io/scheduler-plugins/pkg/networkaware/topologicalsort
# [sigs.k8s.io/scheduler-plugins/pkg/networkaware/topologicalsort]
pkg/networkaware/topologicalsort/topologicalsort_test.go:108:29: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/topologicalsort/topologicalsort_test.go:109:29: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/topologicalsort/topologicalsort_test.go:149:23: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
pkg/networkaware/topologicalsort/topologicalsort_test.go:149:73: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
# sigs.k8s.io/scheduler-plugins/test/integration
# [sigs.k8s.io/scheduler-plugins/test/integration]
test/integration/networkoverhead_test.go:185:22: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
test/integration/networkoverhead_test.go:185:72: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
test/integration/topologicalsort_test.go:148:22: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
test/integration/topologicalsort_test.go:148:72: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
test/integration/topologicalsort_test.go:225:22: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields
test/integration/topologicalsort_test.go:225:72: k8s.io/apimachinery/pkg/apis/meta/v1.Time struct literal uses unkeyed fields

**Comment by ffromani** [comment]
we should perhaps tighten the CI rules after the necessary cleanups are merged (cc @Huang-Wei for awareness)

could you please sign the CLA @maoqide ? it's a prerequisite to be able to contribute. Thanks!

**Comment by maoqide** [comment]
> could you please sign the CLA @maoqide ? it's a prerequisite to be able to contribute. Thanks!

OK. Thanks!

File: pkg/networkaware/networkoverhead/networkoverhead_test.go [hunk 1]
```diff
@@ -286,8 +286,8 @@ func GetAppGroupCROnlineBoutique() *agv1alpha1.AppGroup {
 			},
 		},
 		Status: agv1alpha1.AppGroupStatus{
-			ScheduleStartTime:       metav1.Time{time.Now()},
```
Comment (Huang-Wei) [inline pkg/networkaware/networkoverhead/networkoverhead_test.go:289]: we can simplify this as `metav1.Now()`

**Review by Huang-Wei — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**